Assets/LeapMotion/Modules/InteractionEngine/Scripts/UI/Anchors/AnchorableBehaviour.cs

```diff
@@ -829,15 +829,13 @@ public enum EventType {
       setupCallback(ref OnPostTryAnchorOnGraspEnd, EventType.OnPostTryAnchorOnGraspEnd);
     }
 
-    private void setupCallback<T1, T2>(ref Action<T1, T2> action, EventType type)
-                                       where T1 : AnchorableBehaviour
-                                       where T2 : Anchor                          {
-      action += (anchObj, anchor) => _eventTable.Invoke((int)type);
-    }
-
-    private void setupCallback<T>(ref Action<T> action, EventType type)
-                                  where T : AnchorableBehaviour         {
-      action += (anchObj) => _eventTable.Invoke((int)type);
+    private void setupCallback(ref Action action, EventType type) {
+      if (_eventTable.HasUnityEvent((int)type)) {
+        action += () => _eventTable.Invoke((int)type);
+      }
+      else {
+        action += () => { };
```

Comment:
The callbacks already have a null delegate when they are initialized, can't this else clause be removed?

Reply:
Maybe? This is just the pattern we have in InteractionBehaviour..

Reply:
removed the else clause